source/deployment/enterprise-deployment-guide.rst

```diff
@@ -75,24 +75,28 @@ Further Reading
 Mattermost Scaling
 ------------------
 
-Growing your Mattermost installation from supporting a team to supporting an enterprise requires two types of scaling.
+Growing your Mattermost installation from supporting a team to supporting an enterprise requires two types of `scaling <https://docs.mattermost.com/deployment/scaling.html>`_.
 
-**`Technical Scaling <https://docs.mattermost.com/deployment/scaling.html>`_** - Whether used for teams or enterprises, the Mattermost server is designed to support tens of thousands of users on a single server with appropriate hardware. We officially support running Mattermost Server on multiple `Linux distributions and on-premises cloud solutions <https://docs.mattermost.com/guides/administrator.html#installing-mattermost>`_.
+**Technical Scaling:** Whether used for teams or enterprises, the Mattermost server is designed to support tens of thousands of users on a single server with appropriate hardware. We officially support running Mattermost Server on multiple `Linux distributions and on-premises cloud solutions <https://docs.mattermost.com/guides/administrator.html#installing-mattermost>`_.
 
-**Functional Scaling** - Scaling from a team to an enterprise is like going from a "virtual office" to a "virtual campus". Advanced features like enterprise authentication, granular permissions, compliance and auditing, and advanced reporting become increasingly important as organizations grow beyond teams.
+**Functional Scaling:** Scaling from a team to an enterprise is like going from a "virtual office" to a "virtual campus". Advanced features like enterprise authentication, granular permissions, compliance and auditing, and advanced reporting become increasingly important as organizations grow beyond teams.
 
 Mattermost Licensing
 --------------------
 
-Mattermost Enterprise Edition comes with two licensing options: Enterprise E10 and Enterprise E20 which can be tested with a `Free 30-Day Trial <https://mattermost.com/trial/>`_. The E10 license to suit smaller organizations with less need for automation and compliance, while the E20 license will provide you with features like AD/LDAP, SAML 2.0, automated compliance export, and team-specific permissions. You can find a full comparison `here <https://mattermost.com/pricing-feature-comparison>`_.
+Mattermost Enterprise Edition comes with two licensing options: Enterprise E10 and Enterprise E20.
+
+The E10 license suits smaller organizations with less need for automation and compliance, while the E20 license will provide you with features like AD/LDAP, SAML 2.0, automated compliance export, and team-specific permissions. You can find a full comparison `here <https://mattermost.com/pricing-feature-comparison>`_.
+
+Once you have downloaded and installed Mattermost, navigate to **Main Menu > System Console** to activate a 30-day Enterprise trial.
```

Comment:
```suggestion
Once you have installed and deployed Mattermost, activate your Enterprise E20 trial license via **Main Menu > System Console > Edition and License** and select **Start trial**.
```

Reply:
Reworded for stylistic alignment but not sure it works well. Thoughts?

Reply:
Saying "activate **your** Enterprise E20 trial license" suggests that they already have the trial license, but this feature is all about the simplicity of clicking the **Start trial** button to start the trial and not needing to go get a trial license on your own.

Reply:
So something more along the lines of:

"Start an Enterprise E20 trial via **Main Menu > System Console > Edition and License > Start trial**.

Reply:
@esadur, following up on this. :)

Reply:
Looks fine to me